### ISSUE TITLE
Travis build error during mysql-server install

### ISSUE THREAD

**Issue by zsilbi** [phalcon/cphalcon#14260]
It fails to install  `mysql-server`, so all the new CI builds fail.
See: https://github.com/phalcon/cphalcon/pull/14259

```
The following packages have unmet dependencies:
 mysql-server : Depends: mysql-server-5.5 but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
Looking for 'mysql' as: mysql
Looking for 'mysqlcheck' as: mysqlcheck
This installation of MySQL is already upgraded to 5.6.33, use --force if you still need to run mysql_upgrade
The command ".ci/install-mysql-5.7.sh" failed and exited with 1 during .
```
https://travis-ci.org/phalcon/cphalcon/builds/563215912

**Comment by sergeyklay** [comment]
It is weird that file called install-mysql-5.7.sh tries to install mysql 5.5. @SidRoberts Any idea?

**Comment by SidRoberts** [comment]
I have no idea but I had a similar problem a while back though. The official builds would run fine on Travis but mine would fail - interestingly because of this MySQL script. The problem was because Travis was using different Ubuntu versions. I fixed this in https://github.com/phalcon/cphalcon/pull/14189 by specifying trusty.

It's interesting to note that on this commit, the Phalcon build passed but my build failed:
- https://travis-ci.org/phalcon/cphalcon/jobs/561642481
- https://travis-ci.org/SidRoberts/cphalcon/jobs/561694697

I'll take a look and see what I can remember.

**Comment by SidRoberts** [comment]
Just to provide an update to this, I'm currently working on changing the Ubuntu version used in Travis to 18.04 (bionic). We're currently using 14.04 (trusty) which is woefully outdated and I suspect might be causing some of these problems. It's not complete yet but should be finished today.

**Comment by sergeyklay** [comment]
Fixed in the `4.0.x` branch. Feel free to open a new issue if the problem appears again. Thank you for contributing.
